proxy/pkg/middleware/basic_auth.go

```diff
@@ -89,6 +90,12 @@ func (m basicAuth) isPublicLink(req *http.Request) bool {
 	return ok && login == "public" && strings.HasPrefix(req.URL.Path, publicFilesEndpoint)
 }
 
+// The token auth endpoint uses basic auth for clients, see https://openid.net/specs/openid-connect-basic-1_0.html#TokenRequest
+// > The Client MUST authenticate to the Token Endpoint using the HTTP Basic method, as described in 2.3.1 of OAuth 2.0.
+func (m basicAuth) isOIDCTokenAuth(req *http.Request) bool {
+	return req.URL.Path == "/konnect/v1/token"
```

Comment:
This is now hard-coded for konnectd. What about other IDPs?

Reply:
Other IdPs won't be hidden behind our proxy.

Reply:
I see. Then it should be ok, I guess.